lang/简体中文.json

```diff
@@ -3,19 +3,19 @@
         "terminated": "\n应用程序已终止，请关闭窗口以退出应用程序。",
         "goes_online": "{channel} 该频道已上线, 正在切换...",
         "goes_offline": "{channel} 该频道已离线, 正在切换...",
-        "claimed_drop": "声明的掉宝: {drop}",
-        "claimed_points": "声明的积分奖励: {points}",
+        "claimed_drop": "领取的掉宝: {drop}",
+        "claimed_points": "领取的积分奖励: {points}",
         "earned_points": "观看直播获得积分: {points}, 总积分: {balance}",
         "no_channel": "没有可观看的频道，等待一个在线直播频道...",
         "no_campaign": "没有可用于掉宝的活动，等待一个有效的掉宝活动..."
     },
     "login": {
-        "unexpected_content": "返回的内容类型有误通常是由于被重定向.您需要VPN才能登录吗？",
+        "unexpected_content": "返回的内容类型有误通常是由于被重定向.您需要登陆才能联网吗？",
```

Comment:
Chinese players need to use a VPN to log in, so it is necessary to highlight the VPN here.

Reply:
The original text is about the internet access requiring the login. I'm assuming it's about the public WiFi or hotspots that requires a login or payment, e.g. WiFi in airport or shopping malls, not a bad request due to dns cache pollution. @DevilXD would you mind explaining when will this error notification pop up?

Reply:
If user VPN is not set correctly, the request will timeout, and a "no_connection" error will be shown to tell user that application cannot connect to Twitch. That's why Suz1e added "Check VPN Setting" to the translation of "no_connection" notification. I noticed it and didn't modify his translation to better help user to troubleshoot the issue.

But according to your explaintion, "unexpected_content" is very specific for the unexpected HTML return, which is not caused by GFW dns pollution. So, since the application do received a response from the request, just not the expected ones, I would recommend to direct the user to check the login-like pages instead of checking their VPN setting to better track down the issue.

BTW, most of the public WiFi still need login. Connect to the hotspot, input your phone number and receive one-time code. Then type the code to the login page to be able to use the internet.

Reply:
> But according to your explaintion, "unexpected_content" is very specific for the unexpected HTML return

It specifically happens when anything other than the expected JSON is returned. Not that, while HTML is the most often cause, it's not the only one that could happen.

> BTW, most of the public WiFi still need login. Connect to the hotspot, input your phone number and receive one-time code. Then type the code to the login page to be able to use the internet.

I see. Well, I'm leaving correcting the translation to your hands then :slightly_smiling_face: